docs/docs/04-multiple-components.md

```diff
@@ -180,7 +180,7 @@ In React, data flows from owner to owned component through `props` as discussed
 
 ## A Note on Performance
 
-You may be thinking that it's expensive to react to changing data if there are a large number of nodes under an owner. The good news is that JavaScript is fast and `render()` methods tend to be quite simple, so in most applications this is extremely fast. Additionally, the bottleneck is almost always the DOM mutation and not JS execution and React will optimize this for you using batching and change detection.
+You may be thinking that it's expensive for React to change data if there are a large number of nodes under an owner. The good news is that JavaScript is fast and `render()` methods tend to be quite simple, so in most applications this is extremely fast. Additionally, the bottleneck is almost always the DOM mutation and not JS execution. React will optimize this for you using batching and change detection.
```

Comment:
Both versions are awkward.  Maybe "You may be thinking that it's expensive to change data if there are a large number of nodes under an owner".  Just remove the "to React" altogether, since we're obviously talking about React perf?